packages/service-clients/odsp-client/src/odspClient.ts

```diff
@@ -157,11 +158,11 @@ export class OdspClient {
 			container,
 		});
 		const services = await this.getContainerServices(container);
-		return { container: fluidContainer as IFluidContainer<T>, services };
+		return { container: fluidContainer as IFluidContainer<TreeContainerSchema>, services };
 	}
 
-	private getLoaderProps(schema: ContainerSchema): ILoaderProps {
-		const runtimeFactory = createDOProviderContainerRuntimeFactory({
+	private getLoaderProps(schema: TreeContainerSchema): ILoaderProps {
+		const runtimeFactory = createTreeDOProviderContainerRuntimeFactory({
```

Comment:
Name of this function has changed with [https://github.com/microsoft/FluidFramework/pull/25030](https://github.com/microsoft/FluidFramework/pull/25030). If we want to share this PR further, we should merge from main and update.